### PR TITLE
Add the option to install without kafka / grafana / prometheus

### PR DESCRIPTION
**Pull request by markmark206**
Add the option to install without kafka / grafana / prometheus (e. g. `helm install --set grafana.enabled=false temporaltest .`).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,14 +26,23 @@ Temporal can be configured to run with a couple of database choices.
...
 ```bash
 ~/temporal-helm$ helm install temporaltest . --timeout 900s
 ```
 
+To use your own instance of ElasticSearch, MySQL, or Cassandra, please read the "Bring Your Own" sections below.
+
+Other components can be omitted by setting their corresponding 'enable' flag to `false`, e. g.
+
+```bash
+~/temporal-helm$ helm install --set grafana.enabled=false temporaltest . --timeout 900s
```
Comment (alexshtin) [inline README.md:42]: This looks slightly confusing (at lease to me). `enabled=false` means "disable". One can get impression that after this command his graphana cluster will be disabled. Or at least grphana features in Temporal will be disabled. `graphana.install` looks like a better name for me.

Comment (alexshtin) [reply README.md:42]: It might be applicable to other PRs also.

Comment (markmark206) [reply README.md:42]: oh that's a good point. i think this is a bit of a convention in in helm world to use `enabled` to indicate that the thing is omitted from installation. i'll think on this a bit, to see if there is a better way.

File: Chart.yaml [hunk 2]
```diff
@@ -27,16 +27,19 @@ dependencies:
...
     repository: https://charts.bitnami.com/bitnami
     version: 7.2.9
+    condition: kafka.enabled
   - name: grafana
     repository: https://kubernetes-charts.storage.googleapis.com
     version: 5.0.10
+    condition: grafana.enabled
 
 # A chart can be either an 'application' or a 'library' chart.
 #
```
Comment (kfoxb) [inline Chart.yaml:45]: I think it'd be nice to update to version in this file with this change. To 0.2.0 probably :-)

**Pull request merged**